net/jool/files/jool.init

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh /etc/rc.common
+
+START=54
+STOP=54
+
+config_parser(){
+	local enabled
+	local enabled_nat64
+	local enabled_siit
+
+	config_load "jool"
+
+	#verify if the services are enabled in the configuration and populate it's variables
+	config_get_bool enabled general enabled 0
+	config_get_bool enabled_nat64 nat64 enabled 0
+	config_get_bool enabled_siit siit enabled 0
+
+	#If the main service is not enabled exit
+	[ "$enabled" -eq 0 ] && return 1
+
+	#if nat64 is enabled continue
+	if [[ "$enabled_nat64" -gt 0 ]]; then
+		#check if the orer is to start or stop
+		if [[ "$1" -gt 0 ]]; then
+			#start jool
+			jool file handle /etc/jool/jool-nat64.conf.json
+		else
+			jool -f /etc/jool/jool-nat64.conf.json instance remove
+		fi
+	fi
+
+	#if siit is enabled continue
+	if [[ "$enabled_siit" -gt 0 ]]; then
+		#check if the orer is to start or stop
+		if [[ "$1" -gt 0 ]]; then
+			#start jool
+			jool_siit file handle /etc/jool/jool-siit.conf.json
+		else
+			/usr/bin/jool_siit -f /etc/jool/jool-siit.conf.json instance remove
```

Comment:
interesting that sometimes it's /usr/bin and sometimes the name. Normally a PROG variable is created for this.

Reply:
What do you mean by a PROG variable? I'm not used to creating scripts. 😄

Reply:
In other unit scripts I've seen PROG= at the top and $PROG referenced later on.